CONTRIBUTING.md

```diff
@@ -86,20 +88,18 @@ provided configuration with `pre-commit install`.
 
 Pycharm is an amazing and beautiful python IDE that some of us use to work on this project.
 If you combine it with python black and formatting on save, you will get a very efficient
-workflow.
+workflow. It's also especially efficient for git branching, cherry-picking, committing and pushing.
 
-1. pip install black
-2. Run blackd in a terminal
-3. Install BlackConnect plugin
-4. Set to run python black on save
-5. Set line length to 120
-6. Install the linters in the root directory
+1. Run blackd in a terminal
+2. Install BlackConnect plugin
+3. Set to run python black on save
+4. Set line length to 120
+5. Install the linters in the root directory
```

Comment:
I'm not sure what `5.` means and I personally can't comment on the rest as the closest I come to black integrations is a `ctrl+b` shortcut.  Do you have to have the venv activated in the console where you launch pycharm?

Reply:
not sure what 5 means. I just left it there since I didn't know what it meant